doc: add write_generate_directive flag in README (#59)
uber-go/mock: README.md

- `-write_package_comment`: Writes package documentation comment (godoc) if true. (default true)

> - `-write_generate_directive`: Put the //go:generate directive into the generated code. (default false)

Reviewer: ```suggestion
- `-write_generate_directive`: Add //go:generate directive to regenerate the mock. (default false)
```

Author: @r-hang updated
